### DISCUSSION TITLE
reader specific automations

### DISCUSSION THREAD

**Discussion by shook183** [adonno/tagreader#131] [Q&A] [answered]
hello i had a question about using multiple readers to trigger certain things. for example id like to trigger an automation in the living room with a tag and lets say the tag adjusts the living room to my liking then id like to use the same tag on a different reader upstairs and have that reader automate my bedroom. is creating an event like this possible thank you

**Comment by wtreur** [comment, marked as answer]
You can use the tag reader ID and map it to an entity. Take a look at the jukebox example on https://www.home-assistant.io/blog/2020/09/15/home-assistant-tags/ where it is used to target specific media players

**Comment by teleoltre** [comment]
@shook183 @wtreur Can you give me an example of automation? I'm a beginner

**Comment by wtreur** [comment]
@teleoltre In that case, welcome to home assistant! :) In general, these kind of support questions are better suited to ask on the [Forum](https://community.home-assistant.io/) (although I'm sure @adonno doesn't mind if I help you out a bit here)

Here are some links to get you started with your needs
- [automation support page](https://www.home-assistant.io/docs/automation/basics/)
- [The Automation with YAML section](https://www.home-assistant.io/docs/automation/yaml/) since your need is a bit more advanced
- [This example automation](https://www.home-assistant.io/blog/2020/09/15/home-assistant-tags/#building-the-rfid-jukebox)

Again, feel free to ask for more help on [the various communication channels](https://www.home-assistant.io/help/).

**Comment by dhuddle** [comment]
![image](https://user-images.githubusercontent.com/73569754/199340844-4969b739-176e-4ab8-912d-a01502f2a086.png)

FWIW, this was my quick hacky solution: i just set the condition of "tagreader LED is on" based on which tag reader I wanted to target. So, I can use the same tag (or phone or smart watch) to do different things at different locations.
